src/styles/utils/_mixins.scss

```diff
@@ -1,6 +1,111 @@
+@import "vars";
```

Comment:
Consider using a more descriptive name for the imported file, e.g. '_variables.scss' instead of 'vars.scss'.

Reply:
I changed file to _variables.scss.